x/lazyledgerapp/client/cli/payformessage.go

```diff
@@ -0,0 +1,81 @@
+package cli
+
+import (
+	"encoding/hex"
+	"errors"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	"github.com/lazyledger/lazyledger-app/x/lazyledgerapp/types"
+	"github.com/spf13/cobra"
+)
+
+// CmdCreatePayForMessage returns a cobra command that uses the key ring backend
+// and locally running node to create and broadcast a new WirePayForMessage
+// transaction.
+func CmdCreatePayForMessage() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "payForMessage [hexNamespace] [hexMessage]",
+		Short: "Creates a new WirePayForMessage",
+		Args:  cobra.ExactArgs(2),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			clientCtx, err := client.GetClientTxContext(cmd)
+			if err != nil {
+				return err
+			}
+
+			// get the account name
+			accName := clientCtx.GetFromName()
+			if accName == "" {
+				return errors.New("no account name provided, please use the --from flag")
+			}
+
+			// get info on the key
+			keyInfo, err := clientCtx.Keyring.Key(accName)
+			if err != nil {
+				return err
+			}
+
+			// decode the namespace
+			namespace, err := hex.DecodeString(args[0])
+			if err != nil {
+				return err
+			}
+
+			// decode the message
+			message, err := hex.DecodeString(args[1])
+			if err != nil {
+				return err
```

Comment:
For better UX, maybe you can wrap error and provide info what argument (namespace or message) is wrong?

Reply:
nice catch! You just saved someone some time :slightly_smiling_face: not all heroes wear capes :wink: 

resolved https://github.com/lazyledger/lazyledger-app/pull/32/commits/db5a91c5e1504dbd83dc3e354a5993b6902f8aa9